.changeset/fresh-frogs-heal.md

```diff
@@ -0,0 +1,6 @@
+---
+"@firebase/firestore": patch
+"@firebase/webchannel-wrapper": patch
+---
+
+Fix webchannel wrapper overrides.js to pick useFetchStreams option.
```

Comment:
nit: Re-word this release notes entry into a way that would mean something to a customer reading it. Namely, word it such that it is clear that it is a fix for the bug where the last release incorrectly used XmlHttpRequest instead of Fetch.